tests/e2e/vsphere_file_volume_basic_mount.go

```diff
@@ -154,11 +176,15 @@ Pod2 is created as normal user if secondPodForNonRootUser is true
 Pod1 is deleted if isDeletePodAfterFileCreation is true
 Pod2 is mounted on statically provisioned pvc backed by same file share id as Pvc1 if staticProvisionedPVCForSecondPod true
 */
-func invokeTestForCreateFileVolumeAndMount(f *framework.Framework, client clientset.Interface, namespace string, accessMode v1.PersistentVolumeAccessMode, filePath1 string, filePath2 string, isDeletePodAfterFileCreation bool, secondPodForNonRootUser bool, staticProvisionedPVCForSecondPod bool) {
+func invokeTestForCreateFileVolumeAndMount(f *framework.Framework, client clientset.Interface, namespace string, accessMode v1.PersistentVolumeAccessMode, filePath1 string, filePath2 string, isDeletePodAfterFileCreation bool, secondPodForNonRootUser bool, staticProvisionedPVCForSecondPod bool, fsType bool) {
```

Comment:
`invokeTestForCreateFileVolumeAndMount` is taking 10 arguments and this has reduced the readability of the tests. Can we make use of test specs type and wrap all these arguments as a member in that spec?

If you follow this approach, you will not require to change all existing tests when a new member is added to the testspec type.